database/postgres/postgres.go

```diff
@@ -121,10 +128,12 @@ func (p *Postgres) Open(url string) (database.Driver, error) {
 	}
 
 	migrationsTable := purl.Query().Get("x-migrations-table")
+	statementTimeout, _ := strconv.Atoi(purl.Query().Get("x-statement-timeout"))
```

Comment:
Handle the following:
* `.Get()` returning an empty string
* `error` from `Atoi()`

Reply:
for `error` from `Atoi()` it will return a `0` along with the error in which case we'll use the default timeout not sure if we need to do more there

Reply:
An error should be returned if `Atoi()` returns an error. e.g. a non base 10 integer is specified, silently succeeding would be bad since the user may be unaware of the error and would have the wrong expectations

Reply:
sure we can block execution. And do the same for `.Get()` empty string as well?

Reply:
`
statementTimeout, err := strconv.Atoi(purl.Query().Get("x-statement-timeout"))
if err != nil {
    return nil, err
}
`

This should capture both bullet points. `.Get()` will cause `Atoi()` to error